docs/LocalDTO.md

```diff
@@ -0,0 +1,499 @@
+###LocalMapDTO
+Example:
+ ```json
+{
+	"id":"exampleId",
+	"name":"exampleName",
+	"coordinates":{
+		"lat":12.22,
+		"lon":13.33
+	},
+    "address": {
+        "city": "Warsaw",
+        "street": "Świętokrzyska 4",
+        "postCode": "02-100"
+    },
+    "image": "www.abcimage.com",
+    "openingHours": [
+        {
+            "orderNumber": 1,
+            "dayOfWeek": "Mon",
+            "open": "08:00",
+            "close": "22:00"
+        },
+        {
+            "orderNumber": 2,
+            "dayOfWeek": "Tue",
+            "open": "08:00",
+            "close": "22:00"
+        },
+        {
+            "orderNumber": 3,
+            "dayOfWeek": "Wed",
+            "open": "08:00",
+            "close": "22:00"
+        },
+        {
+            "orderNumber": 4,
+            "dayOfWeek": "Thu",
+            "open": "08:00",
+            "close": "22:00"
+        },
+        {
+            "orderNumber": 5,
+            "dayOfWeek": "Fri",
+            "open": "08:00",
+            "close": "22:00"
+        },
+        {
+            "orderNumber": 6,
+            "dayOfWeek": "Sat",
+            "open": "08:00",
+            "close": "22:00"
+        },
+        {
+            "orderNumber": 7,
+            "dayOfWeek": "MON",
+            "open": "08:00",
+            "close": "22:00"
+        }
+    ],
+    "labels": [
+        {
+            "id": 1,
+            "name": "beer"
+        },
+        {
+            "id": 2,
+            "name": "vodka"
+        }
+    ],
+    "website": "www",
+    "contact": {
+        "phoneNumber": "123123123",
+        "email": "mail@gmail.com"
```

Comment:
what about owner id?

Reply:
IMHO we shouldn't return owner id, its not necessary for frontend purposes